ext/opcache/zend_shared_alloc.h

```diff
@@ -185,7 +185,7 @@ void zend_shared_alloc_destroy_xlat_table(void);
 void zend_shared_alloc_clear_xlat_table(void);
 uint32_t zend_shared_alloc_checkpoint_xlat_table(void);
 void zend_shared_alloc_restore_xlat_table(uint32_t checkpoint);
-void zend_shared_alloc_register_xlat_entry(const void *old, const void *new);
+void zend_shared_alloc_register_xlat_entry(const void *old, const void *new_);
```

Comment:
`new_` I know why you re doing this but I would prefer a proper argument name but just personal taste.

Reply:
Give me a name you like and I'll change it. Name doesn't matter for me, as long as it's not a C++ keyword ;-)

Reply:
I meant without underscore  ; otherwise a name conveying its role s good enough :-)

Reply:
But what is its role? A new .... what? None of this is documented, no code comments anywhere, obscure stuff, I can only try to derive this from the source code.

Reply:
``old_entry`` and ``new_entry``? Although I don't know what this function actually does.

Reply:
The parameters are not two entries - the combination of both is *one entry*. It's some kind of translation table for persisted pointers in the opcache. Or something.

Reply:
Okay trying to stare at it in code usage and looking up what "XLAT" means, maybe something like ``key_derivation_value`` and ``new_value``? If not I'll just pass on trying to find a good name for this.

Reply:
I've renamed the parameter to simply `key` and `value` (in a separate commit). I figured that this is not only used to translate pointers for storing them into shared memory, but also to annotate arbitrary pointers, e.g.
https://github.com/php/php-src/blob/d7f624258de87e8d8e4829ad8f9d279b15ac8162/ext/opcache/jit/zend_jit.c#L4427
I wonder if that kind of usage is an abuse of the API ...